.bazelrc

```diff
@@ -15,12 +15,22 @@ import %workspace%/shared.bazelrc
 # PUBLIC REPO EXCLUSIVE CONFIGS #
 #################################
 
+# Enable bzlmod by default
```

Comment:
These are the defaults in Bazel 8, so this shouldn't be needed.

Reply:
Good call. I probably should move the disabled-bzlmod flag out of the shared.bazelrc file instead.